bin/describe/describe_external_lib_deps.ml

```diff
@@ -36,25 +36,33 @@ module Item = struct
   type t =
     { kind : Kind.t
     ; dir : Path.Source.t
-    ; external_deps : external_lib_dep list
+    ; external_deps : lib_dep list
+    ; internal_deps : lib_dep list
     ; names : string list
     ; package : Package.t option
+    ; extensions : string list
     }
 
   let to_dyn t =
```

Comment:
```suggestion
  let to_dyn {kind; dir; external_deps; internal_deps; names; package; extensions} =
```
This will ensure that the function will stay up to date